Firmware/RTK_Surveyor/Bluetooth.ino

```diff
@@ -147,13 +147,13 @@ void bluetoothStart()
 #ifdef COMPILE_BT
   if (bluetoothState == BT_OFF)
   {
-    char stateName[10];
+    char stateName[11];
```

Comment:
Why the increase by 1?

Reply:
This is a quirk of `stncpy`. I believe it doesn't leave space for a trailing NULL? I'll double-check this.